### DISCUSSION TITLE
About Partition reshuffle function

### DISCUSSION THREAD

**Discussion by JasonLi-cn** [databendlabs/databend#14456] [General]
Why does the reshuffle function not use the consistent Hash algorithm to ensure cache stability?

https://github.com/datafuselabs/databend/blob/ea5eaf097eb65f8ac430a05d40c10a28c495e69f/src/query/catalog/src/plan/partition.rs#L115-L188

**Comment by sundy-li** [comment]
That's right, it could be an improvement.
